src/fswAlgorithms/effectorInterfaces/thrusterPlatformReference/_UnitTest/test_thrusterPlatformReference.py

```diff
@@ -229,6 +237,21 @@ def platformRotationTestFunction(show_plots, delta_CM, K, seed, accuracy):
         testFailCount += 1
         testMessages.append("FAILED: " + platformWrap.ModelTag + "thrusterPlatformReference module failed unit test on thruster torque \n")
 
+    # compare the module results to the python computation for thruster configuration in B frame
+    r_TB_B = r_CB_B - np.matmul(FB.transpose(), r_CT_F)
+    r_TB_B_sim = thrConfigBLog.rThrust_B[0]
+    tHat_B_sim = thrConfigBLog.tHatThrust_B[0]
+    tMax_sim = thrConfigBLog.maxThrust[0]
+    if not unitTestSupport.isVectorEqual(r_TB_B_sim, r_TB_B, accuracy):
```

Comment:
Hit me with some `np.testing` sweetness and then we can merge.

Reply:
On it 🔥